src/EnergyPlus/DElightManagerF.cc

```diff
@@ -237,7 +237,7 @@ namespace DElightManagerF {
 
         // Loop through the Daylighting:Controls objects that use DElight checking for a host Zone
         for (auto &znDayl : state.dataDaylightingData->daylightControl) {
-            if (znDayl.DaylightMethod == DataDaylighting::DaylightingMethod::DElight) {
+            if (znDayl.DaylightMethod == Dayltg::DaylightingMethod::DElight) {
```

Comment:
Truncating this namespace name didn't gain much (i.e., wrapped lines). So why not just use Daylighting::Method here?

Reply:
I didn't even notice this because I do these changes using scripts.